pKa_challenge_instructions.md

```diff
@@ -0,0 +1,115 @@
+
+Three replicate pKa measurements were made for all compounds at room temperature (25°C). Multiwavelength absorbance analysis of Sirius T3 allows very good resolution pKas but essentially this method measures macroscopic pKas. Microscopic pKas with with very close pKa values and overlapping changes in absorbance spectra could be measured as one macroscopic pKa value.
+
+## Instructions and Submission Templates
+Three types of predictions will be accepted. Participants are encouraged to submit their results in all or multiple submission types as it fits to their prediction methods.
+
+#### Type I - microscopic pKas and related microstates
+Predicting microscopic pKa values and related microstates structures. Different protonation states and tautomers constitute different microstates. 
+- Fill the `typeI_microscopic_pKas_and_microstates.csv` template for all molecules.
+- For each molecule, report as many microscopic pKas based on your best guest.
+- Record the pair of microstates IDs of microstate structures pairs (protonated HA and deprotonated A) associated for each microscopic pKa. To determine the microstate ID for your predicted structure check the spreadsheets in [SAMPL6/physical_properties/pKa/microstates](SAMPL6/physical_properties/pKa/microstates).
+- If your predicted structure is not included in the list, contact us to [make a request for new microstate](mehtap.isik@choderalab.org). See more details are in missing microstates section.
+- Report microscopic pKa values with 2 decimals (e.g. 10.71).
+- Reporting standard error of the mean (SEM) is optional and should be reported with 2 decimals. (e.g. 1.02).
+
+#### Type II - microstate populations as a function of pH
+Predicting fractional population of microstates between pH interval 2 to 12 with 0.1 pH increments, based on microscopic pKa predictions.
+- Fill the `typeII_microstate_fractional_populations.csv` template file for all molecules and microstates you have predictions for.
+- To determine the microstate ID for your predicted microstate populations check the spreadsheets in [SAMPL6/physical_properties/pKa/microstates](SAMPL6/physical_properties/pKa/microstates).
+- If your predicted structure is not included in the list, contact us to [make a request for new microstate](mehtap.isik@choderalab.org). See more details are in missing microstates section.
+- Report natural logaritm of fractional microstate populations with scientific notation and 3 decimals of precision, for each pH.  
```

Comment:
"logarithm". And "the natural logarithm", I think.
Also see comment above about decimals of precision.